insta-unfollower.py

```diff
@@ -53,12 +53,16 @@ def login():
         return False
 
     time.sleep(random.randint(2, 6))
-
-    post_data = {
+    if len(sys.argv) > 1:
+        post_data = {
+            'username': sys.argv[1],
+            'enc_password': '#PWD_INSTAGRAM_BROWSER:0:1590954226:' + sys.argv[2]
```

Comment:
Are we sure about this `enc_password` stuff ? maybe it should also be set for environment vars

Reply:
Seemed to work everytime on my side regardless of the username or password